plugins/auth-backend/src/lib/catalog/CatalogIdentityClient.ts

```diff
@@ -71,6 +55,6 @@ export class CatalogIdentityClient {
       }
     }
 
-    return users[0];
+    return (users[0] as unknown) as UserEntity;
```

Comment:
This shouldn't be needed should it? `Entity` should be a subset of `UserEntity`

Reply:
correct, fix'd

won't merge until the other PR that this is on top of is merged tho